### ISSUE TITLE
Metadata is not saved to db in single process mode

### ISSUE THREAD

**Issue by Prometheus3375** [scrapinghub/frontera#396]
Metadata is saved in distributed mode if there is a db worker with no flag `--no-incoming`. When I switched to single process mode, metadata is not saved. I did not find any setting that enables it. Is it indented? According to [documentation](https://frontera.readthedocs.io/en/latest/topics/architecture.html#single-process) metadata should be saved.

![image](https://user-images.githubusercontent.com/35541026/84527871-58797880-ace7-11ea-8866-240c51285c74.png)

[Here](https://github.com/scrapinghub/frontera/files/4772199/crawler.zip) you can find my project. I am using discovery strategy. To add seeds, run `python -m frontera.utils.add_seeds --config config.single --seeds-file seeds.txt`. To start crawl, run `scrapy crawl spider`.

**Comment by sibiryakov** [comment]
Hi, is sqlite file created, is anything written there?

**Comment by Prometheus3375** [comment]
@sibiryakov, yes, queue, states and domain metadata tables are filling with data. Only metadata is always empty in single process mode.
